libs/components/src/messages/components/MessageContent.tsx

```diff
@@ -122,7 +122,7 @@ const MessageContent = memo(
 
     return (
       <Stack width="100%" direction="row">
-        <Box>
+        <Box width="100%">
```

Comment:
why is this needed?

Reply:
Here is what happens without it: when you have a short message, the container doesn't fill the full message column, so the elements are squeezed (the elements only have `max-width`, no `min-width` nor `width`).

<img width="1065" alt="image" src="https://github.com/Chainlit/chainlit/assets/494686/3707635f-e1cc-464a-a479-a416cc21057b">

Reply:
With the change:

<img width="1086" alt="image" src="https://github.com/Chainlit/chainlit/assets/494686/7ac2c273-0d06-4eef-b05a-68396e8be7d5">